bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/provider/ShellyChannelDefinitions.java

```diff
@@ -669,8 +669,8 @@ public ShellyChannel(ShellyTranslationProvider messages, String group, String ch
                 label = "";
             }
             description = getText(PREFIX_CHANNEL + typeId + ".description");
-            if (description.contains(PREFIX_CHANNEL)) {
-                description = "";
+            if (description.startsWith(PREFIX_CHANNEL)) {
```

Comment:
Why is this fix only needed here and not for the other similar lookups above?

Reply:
Please do not mark comments as resolved without either providing feedback or a commit addressing the comment. For this comment I see neither, so I'm reopening. In line 668 (for example) you have exactly the same `contains` check. Why is it changed here, but not there?

Reply:
hmm, one the one side startsWith is correct, because it's a prefix. However, as you outlined the are multiple of those compares - also for group name. I think I should change all of them.

I would like to merge it as is and then m are the changes as part of the next PR. Do you agree?

Reply:
Is the change related to one of the linked bugs? Wouldn't it be more consistent to fix all or none of them?

Reply:
ok, I removed it for this one and rebase should bring it with the next PR

Reply:
@jlaur could you please give feedback here so @lolodomo could click the merge button, would be very helpful so I could move on to the next PR, doing the rebase and we could work on the merge. In fact it is already completely reviewed, because this was is a subset. Nevertheless, there might be some details I need to fix.